scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -34,10 +34,23 @@
       }
 
       .ui-tabs-panel {
+        display: table;
+        overflow-x: auto;
         padding: 0 !important;
+        width: 100%;
 
-        table:first-child {
+        // Makes sure any element inside the panel has a true width 100% depending
+        // on the widest sibling and not the panel's width:
+        > * {
+          display: table-row;
+        }
+
+        * table:first-child {
```

Comment:
same as above for `*`, why is it needed? Doesn't the table have always the same parent element?